wiki/Medals/en.md

```diff
@@ -46,8 +48,6 @@ As the Hush-Hush medals are meant to be secret achievements of some sort, the un
 
 Although the Monthly Beatmap Spotlights have been discontinued as of April 2018 in favour of the [Seasonal Spotlights](https://osu.ppy.sh/home/news/2018-11-01-beatmap-spotlights-summer-2018), all of the medals pertinent to it are still able to be obtained to this day.
 
-## Seasonal Spotlights
-
 **Seasonal Spotlights medals** are medals awarded upon completion of certain beatmap packs released as part of the [Seasonal Spotlights](https://osu.ppy.sh/home/news/2018-11-01-beatmap-spotlights-summer-2018). In order to unlock the medals in this category, a player needs to play through all the maps *in any of the four game mode packs* that are being contested on the corresponding Seasonal Spotlights ranking chart. The following mods are permitted for use:
```

Comment:
this needs to be reworded since there's no distinction anymore. they're all called "Beatmap Spotlights" medals